Packs/CommunityCommonDashboards/Scripts/XMetrics/XMetrics.py

```diff
@@ -83,14 +83,14 @@ def BuildEffortReductionTable(incidents, efforts) -> dict:
         for month, val in v.items():
             ival = int(val)
             if inctype in efforts:
-                autoeffort = int(ival * efforts[inctype][1])
+                deleff = efforts[inctype][0] - efforts[inctype][1]
```

Comment:
Can you please explain the new calculation? (you can make a video that explains it)